hdf5-vol-daos.spec

```
@@ -1,7 +1,7 @@
 %global with_mpich 1
 %global with_openmpi3 1
 
-%global daos_major 0
+%global daos_major 1
```

The libdaos major version bump has not landed to master.
```suggestion
%global daos_major 0
```